### ISSUE TITLE
About the learning rate

### ISSUE THREAD

**Issue by YeDeming** [allenai/document-qa#22]
Hi Christopher,

I feel strange about the high learning rate in ablate_triviaqa.py. I find the learning rate is still 1 and it won't decay in training process. 

I am confused about why do you use the high learning rate, and why do you use so many epochs(71 epochs) in sigmoid convince method with the high learning rate.

Does the lower learning rate,likes 0.01, work?

Thanks a lot for your time!
YeDeming

**Comment by chrisc36** [comment]
Using a learning of 1.0 is standard practice for the Adadelta optimizer, which dynamically adjusts the learning rate anyway. I suspect using different learning rates will not make a large difference.

We use many epochs for the sigmoid method because it took a long time to converge in my experiments.

**Comment by YeDeming** [comment]
Thanks a lot for your help!

**Comment by YeDeming** [comment]
Hi Christopher,

I still have a small question: 

In Class BoundaryPrediction at docqa/nn/span_prediction.py,  in function get_best_span(), I find you convert start_logits and end_logits to function best_span_from_bounds().

As far as I am concerned, we should convert tf.nn.log_softmax(start_logits) &&  tf.nn.log_softmax(end_logits) rather than start_logits&end_logits ?

Thanks a lot for your time!
YeDeming
